kolibri/core/assets/src/views/MenuNav.vue

```diff
@@ -228,7 +272,7 @@
   ];
 
   export default {
-    name: 'SideNav',
+    name: 'MenuNav',
```

Comment:
You have renamed the component, but this component contains strings, this breaks string freeze. Will need to revert the rename here.